marlowe-playground-client/src/MainFrame.purs

```diff
@@ -254,8 +255,12 @@ handleAction (LoadMarloweScript key) = do
   case Map.lookup key StaticData.marloweContracts of
     Nothing -> pure unit
     Just contents -> do
-      marloweEditorSetValue contents (Just 1)
-      updateContractInState contents
+      let
+        prettyContents = case runParser (parseTerm P.contract) contents of
+          Right pcon -> show $ pretty pcon
+          Left _ -> contents
+      marloweEditorSetValue prettyContents (Just 1)
+      updateContractInState prettyContents
```

Comment:
previously the demo contracts were loaded in the format that they were generated with from Haskell. This is not the same as the pretty printer we use here so the first time you changed something the whole shape of the contract would change which is confusing.